lib/main_common.pm

```diff
@@ -1493,7 +1493,7 @@ sub load_extra_tests_textmode {
     }
     loadtest "console/cron";
     loadtest "console/syslog";
-    unless (is_sle '<15') {
+    if (sle_version_at_least('15') && !is_jeos) {
```

Comment:
Please avoid the deprecated method "sle_version_at_least"

Reply:
I'm not aware that it's deprecated - it's used all over in this file...

Reply:
yep, you could help me with https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/5437 :)

Reply:
yes it is still used a lot in our codebase , but a lot of people are eager to drop it

Reply:
What's the equivalent - `is_sle('>=15')"?

Reply:
yes